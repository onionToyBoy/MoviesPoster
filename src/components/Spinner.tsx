import React from 'react';

import styles from './Spinner.module.css';

export const Spinner: React.FC = () => {
	return (
		<div className={styles.container}>
			<div className={styles.spinner}>
				<div className={styles.loader}>Loading...</div>
			</div>
		</div>
	);
};
