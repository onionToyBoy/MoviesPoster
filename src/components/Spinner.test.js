import React from 'react';
import {Spinner} from './Spinner';
import {shallow} from 'enzyme';

describe('Spinner test', () => {

	test('Should render Spinner component correct', () => {
		const component = shallow(<Spinner/>);
		expect(component).toMatchSnapshot();
    });
});