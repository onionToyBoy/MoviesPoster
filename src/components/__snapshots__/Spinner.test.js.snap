// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner test Should render Spinner component correct 1`] = `ShallowWrapper {}`;
